checkQC/handlers/reads_per_sample_handler.py

```diff
@@ -32,12 +32,19 @@ def check_qc(self):
         for lane_dict in self.conversion_results:
             lane_nbr = int(lane_dict["LaneNumber"])
             lane_demux = lane_dict["DemuxResults"]
-            nbr_of_samples = len(lane_demux)
+            total_reads = {}
```

Comment:
I would suggest using `defaultdict` here to simplify the code a bit. I.e.
```
from collections import defaultdict
total_reads = defaultdict(0)
```
Then you can omit the if statement and only do `total_reads[sample_name] += sample_id_info["NumberReads"] / pow(10, 6)`.

Reply:
Cool, will do!